library/oraclelinux

```diff
@@ -4,10 +4,10 @@ GitCommit: 5c8a1c296acd6e90487cd261d16cf85fd6bcb73f
 GitFetch: refs/heads/master
 # https://github.com/oracle/container-images/tree/dist-amd64
 amd64-GitFetch: refs/heads/dist-amd64
-amd64-GitCommit: 2fec92641c29d7a4cc704014d081fa42816ec82f
+amd64-GitCommit: d2cddc02be12d3574ddf2553d94d62ef2d906db1
 # https://github.com/oracle/container-images/tree/dist-arm64v8
 arm64v8-GitFetch: refs/heads/dist-arm64v8
-arm64v8-GitCommit: 0b869f1e3a5cb823ace5ac25a0624e63d4d40f7a
+arm64v8-GitCommit: d2cddc02be12d3574ddf2553d94d62ef2d906db1
```

Comment:
I think this one is supposed to be https://github.com/oracle/container-images/commit/a081765cbdaf3306c7f1ccc45dd525cea67a96b5 (which is why the CI is complaining about the commit not being in the specified branch).

Reply:
Thanks, cut-and-paste mouse-buffer issue.  I have fixed it now.